### ISSUE TITLE
EVE SSO

### ISSUE THREAD

**Issue by trichner** [jeremieroy/EVELiveIntel#2]
Heya, nice tool.

Very nice tool you built here, have you considered adding EVE SSO for authentication?
If you want to add it I might be able to help you, I built the following Incursions Tool for E-Uni https://github.com/trichner/nemesis with Eve SSO.

Cheers, a fellow unista.

**Comment by trichner** [comment]
Okay, I gave it a try, mostly it works now with an entirely new backend but still has some bugs in the frontend. 
Updated version can be found here:
https://github.com/trichner/eve-live-intel

I would really appreciate some help updating your fancy frontend.

**Comment by jeremieroy** [comment]
I apologize, I didn't see your message until today :s
The authentication in place in this repo should be considered a placeholder for the one you want to use in your corp... big corporation often have their own identification framework.
I strongly considered EVE SSO but it wasn't ready when I made the map and... the EveUni identification framework was a W.I.P. at the time as well, maybe it is out now.
I stopped playing in the meantime due to lack of time (job and family...).

"ill has some bugs in the frontend"
What is the issue ?
